src/templates/html/application/ApplicationHTML.tsx

```diff
@@ -283,14 +287,13 @@ export function ApplicationHTML(props: { data: PlanXExportData[] }) {
               {boundary && (
                 <Box sx={{ marginBottom: 1 }}>
                   <my-map
-                    staticMode={true}
                     showNorthArrow={true}
                     showScale={true}
                     useScalebarStyle={true}
-                    hideResetControl={true}
```

Comment:
so we actually still want `hideResetControl={true}` here I think as this only hides the reset button / the geojson should always be static on an overview document unlike an active drawMode map! We just want to remove `staticMode` which then displays the zoom controls again.